app/controllers/admin/protocol_drugs_controller.rb

```diff
@@ -2,25 +2,44 @@ class Admin::ProtocolDrugsController < AdminController
   before_action :set_protocol
   before_action :set_protocol_drug, only: [:show, :edit, :update, :destroy]
 
+  skip_after_action :verify_authorized, if: -> { Flipper.enabled?(:new_permissions_system_aug_2020, current_admin) }
+  skip_after_action :verify_policy_scoped, if: -> { Flipper.enabled?(:new_permissions_system_aug_2020, current_admin) }
+  after_action :verify_authorization_attempted, if: -> { Flipper.enabled?(:new_permissions_system_aug_2020, current_admin) }
+
   def index
-    authorize([:manage, ProtocolDrug])
-    @protocol_drugs = policy_scope([:manage, ProtocolDrug])
+    if Flipper.enabled?(:new_permissions_system_aug_2020, current_admin)
+      authorize1 { current_admin.power_user? }
```

Comment:
This is a good first guess, let's figure out who should be allowed to manage protocol drugs.

Praveen will not be a power user, right? Nilenso will have to manage protocol drugs if we ship this?

Reply:
Yup, would limiting this to organization managers work? Protocols are basically independent.

Reply:
Yeah it's kinda weird, eh? I really like the idea of `authorize1 { current_admin.accessible_organizations(:manage).any? }`. If you think that makes sense, let's do that right away 👌 

If we're not sure, let's leave as is and revisit later. In the interim, we'll manage protocols 😅

Reply:
I've done this, and reused the idea in the `accessible_protocols` and `accessible_protocol_drugs` implementations.